### ISSUE TITLE
Unable to get items from parsing schema

### ISSUE THREAD

**Issue by cruepprich** [schunt1/ATAF#6]
Hi,
I've installed ATAF in a schema named "ATAF". I installed the MV agents in the parsing schema "CMR" of my sample app. I've issued the necessary grants and updated the MVs in ATAF and CMR.

My sample application id is 101. My ATAF app id is 108.

When checking the ataf.ataf_apex_page_items, I still only see those of the ATAF app (108).  I'm not sure what I'm doing wrong at this point.

**Comment by schunt1** [comment]
Yes, it can be a challenge to begin with as it all depends on how you have your environments set up. 

There are 4 x Materiallzed Views that query the APEX views and underpin ATAF, these are:
ATAF_AGENT_PAGE_ITEMS_MV
ATAF_AGENT_PAGES_MV
ATAF_AGENT_SHORTCUTS_MV
ATAF_AGENT_THEMES_MV
If you query these, then I expect they only contain data from application 108, hence why you’re only seeing one application in ATAF. 

1.  Try refreshing ATAF_AGENT_PAGE_ITEMS_MV from SQL Plus connected as ATAF (I expect this will not change anything, but sometimes works)
2.  Try refreshing ATAF_AGENT_PAGE_ITEMS_MV from the SQL Workshop in APEX (this runs in another context and may give different results)
2.  Try refreshing the views from the application. ("Click to Refresh” at the foot of the Interactive Report on page 1).  This will refresh in the context of the application, which might bring the other applications in.
3.  If that doesn’t work, you will need to install agent views into the parsing schema of application 101.  This is explained in the install doc and basically involves applying grants between the schemas and adding them to the ATAF views.

It’s a bit trial and error, but once you crack it you’ll know what works for you.

Let me know how you get on.

**Comment by cruepprich** [comment]
Thanks, Simon. Refreshing from the SQL Workshop did the trick!
